packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -15,12 +15,21 @@ interface PatientBannerProps {
   patient: Pick<fhir.Patient, 'id' | 'name' | 'gender' | 'birthDate' | 'identifier' | 'address' | 'telecom'>;
   patientUuid: string;
   onClick?: (patientUuid: string) => void;
+  closePatientSearchResultsPanel?: () => void;
 }
 
-const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, onClick }) => {
+const PatientBanner: React.FC<PatientBannerProps> = ({
+  patient,
+  patientUuid,
+  onClick,
+  closePatientSearchResultsPanel,
+}) => {
   const { t } = useTranslation();
   const overFlowMenuRef = React.useRef(null);
-  const state = React.useMemo(() => ({ patientUuid, onClick }), [patientUuid, onClick]);
+  const state = React.useMemo(
```

Comment:
I'm just noticing that we are using this same "state" object for both `patient-photo-slot` and `patient-actions-slot`. Can we split it, so that each slot is just receiving the minimum props?